docs/Acceptable-Formulae.md

```diff
@@ -13,10 +13,10 @@ We now accept versioned formulae as long as they [meet the requirements](Version
 
 ### We don’t like tools that upgrade themselves
 Software that can upgrade itself does not integrate well with Homebrew's own
-upgrade functionality. The self-update functionality should be disabled if at all possible.
+upgrade functionality. The self-update functionality should be disabled if at all possible (which it nearly always is).
```

Comment:
What I was trying to say is that while it may be possible, that's not the standard. We intentionally allow some things to auto update that could be made not to auto update like composer and wp-cli.

Reply:
@ilovezfs Yeh, I was thinking of them too. Suggested rewording?

Reply:
Maybe "if possible without complicating the formula" where "complicating the formula" is defined as "doing things ilovezfs finds objectionable."